components/brave_wallet/browser/keyring_controller.h

```diff
@@ -168,6 +172,11 @@ class KeyringController : public KeyedService, public mojom::KeyringController {
 
   void AddAccountForDefaultKeyring(const std::string& account_name);
 
+  // Address will be returned when success
```

Comment:
absl::optional string return for failure?

Reply:
fixed in 8ef138a5650682db54f7b63468334df477ed4d37